presto-main/src/main/java/com/facebook/presto/operator/Driver.java

```diff
@@ -20,7 +20,9 @@
 import com.facebook.presto.spi.UpdatablePageSource;
```

Comment:
Add a short comment in the commit message explaining how it's being simplified (i.e., what previous assumption we were making that's unnecessary, etc)